library/libraries.js

```diff
@@ -1797,6 +1797,26 @@ var d41d8cd98f00b204e9800998ecf8427e_LibraryDetectorTests = {
             if (win.__GUESS__ && win.__GUESS__.guess) {
                 return { version: UNKNOWN_VERSION };
             }
+            return false;
+        }
+    },
+    'October CMS': {
+        id: 'octobercms',
+        icon: 'octobercms',
+        url: 'https://octobercms.com/',
+        npm: null,
+        test: function (win) {
+            const generatorMeta = document.querySelector('meta[name="generator"][content^="OctoberCMS"]');
+
+            // October CMS resource patterns / paths - search in link, style or script tags
+            const resourcesOctober = /\/modules\/system\/assets\/(css|js)\/framework\.(extras|combined)-min/;
+            const res = Array.from(document.querySelectorAll('link,style,script') || []);
+
+            if (generatorMeta || res.some(s => resourcesOctober.test(s.src)) || res.some(s => resourcesOctober.test(s.href))) {
```

Comment:
```suggestion
            if (generatorMeta || res.some(s => resourcesOctober.test(s.src || s.href))) {
```

Would this work too?